slp2mp4/User/Config/GFX.ini

```diff
@@ -69,14 +69,14 @@ ShowFrameTimes = False
 [Enhancements]
 ForceFiltering = False
 DisableFiltering = False
-MaxAnisotropy = 4
+MaxAnisotropy = 1
 PostProcessingEnable = False
 PostProcessingTrigger = 0
 PostProcessingShaders = 
 ScalingShader = 
 UseScalingFilter = True
 TextureScalingType = 0
-TextureScalingFactor = 2
+TextureScalingFactor = 1
```

Comment:
Reduce some default effects to improve performance which can be better used by multiprocessing.